.github/workflows/start-release.jsonnet

```diff
@@ -562,6 +562,11 @@ local notify_failure_job = {
   on: {
     workflow_dispatch: input,
   },
+  // These extra permissions are needed by some of the jobs, e.g. check-semgrep-pro.
```

Comment:
Can you factorize those and put them in gha.libsonnet, next to the write_permissions field
and explain what each field mean and why it's different from the other write_permissions in gha.libsonnet.

Reply:
Looks like start-release actually needs write permission, so write_permission should be used here.  I'll send a follow up PR soon.